args.go

```diff
@@ -468,6 +468,15 @@ const (
 	// ArgVPCIPRange is a VPC range of IP addresses in CIDR notation.
 	ArgVPCIPRange = "ip-range"
 
+	// ArgVPCPeeringName is a name of the VPC Peering.
+	ArgVPCPeeringName = "name"
+	// ArgVPCPeeringFirstVPCID is the first vpc id of the peering
+	ArgVPCPeeringFirstVPCID = "first-vpc-id"
```

Comment:
can this arg take a list of VPC IDs as in "uuid-1, uuui2"

Reply:
yeah we can make it. I'm not sure whether it looks good.  @lfundaro-do @brunograsselli wdyt?

Reply:
yeah, that could work, I like Viola's idea

Reply:
feedback is incorporated